core/trino-spi/pom.xml

```diff
@@ -885,6 +885,12 @@
                                     <old>method void io.trino.spi.security.SystemSecurityContext::&lt;init&gt;(io.trino.spi.security.Identity, java.util.Optional&lt;io.trino.spi.QueryId&gt;)</old>
                                     <new>method void io.trino.spi.security.SystemSecurityContext::&lt;init&gt;(io.trino.spi.security.Identity, io.trino.spi.QueryId, java.time.Instant)</new>
                                 </item>
+                                <item>
+                                    <ignore>true</ignore>
+                                    <code>java.method.numberOfParametersChanged</code>
```

Comment:
You can avoid that easily. Just leave old getInfo as deprecated and add new one delegating back to the old.

Reply:
Yes, please leave the old one with `@Deprecated(forRemoval = true)`